server/src/models/forests.es6

```diff
@@ -8,5 +8,9 @@ forestsDb.Sequelize = Sequelize;
 forestsDb.sequelize = sequelize;
 
 forestsDb.fsForests = require('../models/fs-forests.es6')(sequelize, Sequelize);
+forestsDb.firewoodPermits = require('../models/firewood-permits.es6')(sequelize, Sequelize);
+
```

Comment:
Unnecessary blank line can be removed

Reply:
eslint requires a blank like after a require not followed by another require

Reply:
> eslint requires a blank like after a require not followed by another require

my fault, sorry for wasting your time on this one Mike!